testing/tests/DevExpress.ui.widgets.scheduler/recurrenceEditor.tests.js

```diff
@@ -496,6 +496,14 @@ module('Repeat-on editor', repeatOnModuleConfig, () => {
         assert.equal($repeatOn.length, 1, 'repeat-on editor was rendered');
     });
 
+    test('Recurrence repeat-on lablle should have correct css class', function(assert) {
+        this.createInstance({ value: 'FREQ=WEEKLY' });
+
+        const $repeatOnLabel = this.instance.$element().find('.' + REPEAT_ON_EDITOR + LABEL_POSTFIX);
```

Comment:
Using of interpolation is more preferable: `.${REPEAT_ON_EDITOR}${LABEL_POSTFIX}`